client/src/app.js

```diff
@@ -27,6 +28,8 @@ export const firebaseClient = new FirebaseClient({
   },
 });
 
+export const bugsnag = new Client();
```

Comment:
protip: put this in it's own file `bugsnag.js` then import from there so it's separate from the rest of the intermingled garbage in `app.js`